docs/src/main/tut/json.md

```diff
@@ -177,8 +177,8 @@ val http4s = repos("http4s")
 
 val stargazers = http4s.map(_.map(_.stargazers_count).mkString("\n"))
 
-// Run has been separated into its own line for tut to compile without hanging. 
-stargazers.run
+// UnsafePerformSync has been separated into its own line for tut to compile without hanging.
```

Comment:
... could you make this lowercase? I think in this case it trumps the "sentence starts with capital letter".